test.sh

```diff
@@ -98,6 +98,13 @@ check_fastmath()
     echo "Checking Missing fastmath flags in njit functions"
     ./fastmath.py --check stumpy
     check_errs $?
+
+    echo "Checking hardcoded fastmath flags in njit functions"
+    if [[ $(grep -n fastmath= stumpy/*py | grep -vE 'fastmath=config' | wc -l) -gt "0" ]]; then
+        grep -n fastmath= stumpy/*py | grep -vE 'fastmath=config'
```

Comment:
@seanlaw 
Although I understand that a comment might be added to one of modules in the future that contains the string `fastmath=` (which can result in raising error when running test), I think it should be okay to not consider such scenario to keep the shell script code simple. 

What do you think?

Reply:
In that particular case, we'd catch it BEFORE it would be committed to the codebase since tests will fail so it's not an issue at all.

Reply:
In other words, our current check already covers that case and simply disallows it. So the comment would need to be written differently if it were to be included. If it's absolutely necessary then we will eventually need to update the `grep` expression but, for now, it isn't necessary.

Reply:
> we'd catch it BEFORE it would be committed to the codebase
>  So the comment would need to be written differently if it were to be included

EXACTLY! All good